README.md

```diff
@@ -53,15 +52,13 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: 'Check Jira ticket for two mandatory fields'
-        uses: im-open/verify-fields-on-jira-task@v1.0.2
+        uses: im-open/verify-fields-on-jira-task@v1.0.3
         with:
           domain-name: 'jira.company.com'
-          project-names-to-search: 'First Project, Second Project'
-          issue-types-to-search: 'Task, Subtask'
-          search-field-id: '12345'
-          search-value: 'my-repo/releases/tag/v1.0.0'
-          fields-to-check-for-values: 'customfield_2345, customfield_3456'
+          project-names-to-search: 'My Jira Project'
+          jira-ticket: 'ABCD-1234'
           check-parent-task: 'true'
```

Comment:
`check-parent-task: 'true'` Awesome, so we just provide the deployment task ticket and this searches both. That's cool